Rick&Morty/RickAndMorty/Scenes/Episodes/EpisodeList/EpisodeRowItemView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  EpisodeRowItemView.swift
+//  RickAndMorty
+//
+//  Created by Oliver Jansta on 12.08.2022.
+//  Copyright © 2022 STRV. All rights reserved.
+//
+
+import SwiftUI
+
+struct EpisodeRowItemView: View {
+    let episode: Episode
+    
+    var body: some View {
+        HStack {
+            VStack(alignment: .leading, spacing: 8) {
+                Text(episode.name)
+                    .font(.appItemLargeTitle)
+                    .foregroundColor(.appTextItemTitle)
+                Text("\(episode.code) - \(episode.airDate.formatted(date: .long, time: .omitted))")
```

Comment:
I would prefer an empty line between the SwiftUI Views in general (in this case `Text`s) but that's just my preferred code style ☺️